client/app/bundles/course/survey/pages/ResponseShow/ResponseForm.jsx

```diff
@@ -11,6 +11,10 @@ const styles = {
   submitButton: {
     marginRight: 10,
   },
+  unsubmitButton: {
+    marginRight: 10,
+    backgroundColor: 'red',
```

Comment:
Sorry, missed this earlier. Could you `import { red500 } from 'material-ui/styles/colors';` and do `backgroundColor: red500`?